### ISSUE TITLE
How to call libdarklib.a from python?

### ISSUE THREAD

**Issue by agirbau** [AlexeyAB/darknet#2727]
Hello!

After the changes in CMake I see that libdarknet.so is no longer generated. Before I used that library to execute darknet from python. Do you know how to use darknet from python with this changes? Thank you very much!

**Comment by AlexeyAB** [comment]
@agirbau Hi,

* Try to change `add_library(darklib `  to `add_library(darklib SHARED ` here https://github.com/AlexeyAB/darknet/blob/9815012a01c13f70806381e0c5d556599117c76b/CMakeLists.txt#L250

* Remove `CMakeCache.txt` and `Makefile`

* And recompile Darknet again:

**Comment by agirbau** [comment]
It works perfectly, thank you very much!

**Comment by Fetulhak** [comment]
> @agirbau Hi,
> 
> * Try to change `add_library(darklib `  to `add_library(darklib SHARED ` here https://github.com/AlexeyAB/darknet/blob/9815012a01c13f70806381e0c5d556599117c76b/CMakeLists.txt#L250
> * Remove `CMakeCache.txt` and `Makefile`
> * And recompile Darknet again:

Hi @AlexeyAB @agirbau @cenit sorry to bother you on a closed issue. I am new to darknet and I have got the following error message when I try to run darknet.py. I have followed the above step but still I am getting the error message. Any suggestions will be a great support. I want to get an inference output using the darkent,py

.
.
.
----> 4         netMain = darknet.load_net_custom(configPath.encode( "ascii"), weightPath.encode("ascii"), 0, 1)  # batch size = 1
      5 
      6         metaMain = darknet.load_meta(metaPath.encode("ascii"))

AttributeError: module 'darknet' has no attribute 'load_net_custom'
